quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMappers.java

```diff
@@ -162,6 +322,29 @@ public Map<OptionCategory, List<PropertyMapper<?>>> getRuntimeMappers() {
         public Map<OptionCategory, List<PropertyMapper<?>>> getBuildTimeMappers() {
             return buildTimeMappers;
         }
-    }
 
+        public Map<String, PropertyMapper<?>> getDisabledBuildTimeMappers() {
+            return disabledBuildTimeMappers;
+        }
+
+        public Map<String, PropertyMapper<?>> getDisabledRuntimeMappers() {
+            return disabledRuntimeMappers;
+        }
+
+        private static void sanitizeMappers(Map<OptionCategory, List<PropertyMapper<?>>> mappers,
+                                            Map<String, PropertyMapper<?>> disabledMappers) {
+            mappers.forEach((category, propertyMappers) ->
+                    propertyMappers.removeIf(pm -> {
+                        final boolean shouldRemove = !pm.isEnabled();
+                        if (shouldRemove) {
+                            MAPPERS.removeMapper(pm);
+                            disabledMappers.put(pm.getTo(), pm);
```

Comment:
just a nit, we could consider a function to handle this and for the other case - static void addMapper(PropertyMapper mapper, BiConsumer<String, PropertyMapper> adder) ...

Reply:
Done here:
https://github.com/keycloak/keycloak/pull/25333/files#diff-a1bb1ef613469cf669070d52be8a88079005dcaed860e8d11d4231b5e2f530f7R343-R351